poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
```

Comment:
@cptvitamin Thanks for pointing this out -- this was a bug in one of the Django templates, resulting in `None` to be shown instead of the empty string.  (We still have to show an empty string if no alternate description is available, e.g. for block usages that have been created before it was possible to add alt tags.)